pages/en/about/branding.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Branding of Node.js
+layout: about.hbs
+---
+
+# Branding of Node.js
+
+Please review the [trademark policy](https://trademark-policy.openjsf.org/) for information about permissible use of Node.js® logos and marks.
+
+## Node.js® Mascot
+
+<img
+  alt="Node.js mascot"
+  src="/static/images/node-mascot.svg"
+  className="w-[100px]"
```

Comment:
In places where we use the `img` tag, we should also use `width` and `height` attributes to avoid causing [CLS](https://web.dev/articles/optimize-cls?utm_source=lighthouse&utm_medium=devtools#images-without-dimensions). We can also consider using [Next.js's Image component](https://nextjs.org/docs/pages/api-reference/components/image) in mdx files.

Reply:
+1 for nextjs image. 
I'll take a look for `width` and `height`

Reply:
> We can also consider using Next.js's Image component in mdx files 👀

We do not? Oh no! We should add the mapping here: https://github.com/nodejs/nodejs.org/blob/main/next.mdx.use.mjs#L92-L99

@canerakdas do you have time to make a PR adding the Img component there and checking if builds passes?